pkg/kudoctl/packages/types.go

```diff
@@ -43,9 +64,152 @@ func (p Parameter) Less(x, y int) bool {
 	return p[x].Name < p[y].Name
 }
 
+// Templates is a map of file names and stringified files in the template folder of an operator
+type Templates map[string]string
```

Comment:
We discussed it briefly offline, so here a **tl;dr**:
- `Templates`, `Nodes` and `TemplateNode` are an implementation detail of the `/verify` package and doesn't have to be exposed
- The entry point of the verification code should probably take `package.Files` and return a set of errors/warnings.